test/ZyRabbit.PerformanceTest/MessageContextBenchmarks.cs

```diff
@@ -51,30 +45,15 @@ public void Cleanup()
 		[Benchmark]
 		public async Task MessageContext_FromFactory()
 		{
-			var msgTsc = new TaskCompletionSource<Message>();
-
-			EventHandler onMessageReceived = (sender, args) => { msgTsc.TrySetResult(sender as Message); };
-			MessageReceived += onMessageReceived;
-
-			_withContext.PublishAsync(_messageB);
-			await msgTsc.Task;
-			MessageReceived -= onMessageReceived;
+			await _withContext.PublishAsync(_messageB);
```

Comment:
why remove awaiting the message to be received?

Reply:
fair point, i reverted this code